src/arch/linux/Makefile.main

```diff
@@ -211,7 +211,11 @@ install:
 	done
 	$(INSTALL) -d $(DESTDIR)$(docdir)
 	for i in README.bindist NEWS THANKS BUGS changelog; do \
-	  $(INSTALL) -m 0644 $(REALTOPDIR)/$$i $(DESTDIR)$(docdir); \
+	  if [ -f $(REALTOPDIR)/$$i ]; then \
+	    $(INSTALL) -m 0644 $(REALTOPDIR)/$$i $(DESTDIR)$(docdir); \
+	  else \
+	    $(INSTALL) -m 0644 ../$$i $(DESTDIR)$(docdir); \
+	  fi; \
```

Comment:
Would it be possible to do without conditions?
Maybe by using some other var than `REALTOPDIR`?